composer/trainer/trainer.py

```diff
@@ -289,6 +291,39 @@ def _generate_run_name() -> str:
     return generated_run_name
 
 
+def _maybe_create_remote_uploader_downloader_from_uri(
+        uri: str, loggers: List[LoggerDestination]) -> Optional[RemoteUploaderDownloader]:
+    existing_remote_uds = [logger_dest for logger_dest in loggers if isinstance(logger_dest, RemoteUploaderDownloader)]
+    backend, bucket_name, _ = _parse_uri(uri)
+    if backend == '':
+        return None
+    for existing_remote_ud in existing_remote_uds:
+        if ((existing_remote_ud.remote_backend_name == backend) and
+            (existing_remote_ud.remote_bucket_name == bucket_name)):
+            warnings.warn(
+                f'There already exists a RemoteUploaderDownloader object to handle the uri: {uri} you specified')
```

Comment:
I think this is `info` not `warning`? The user doesn't need to do anything, and this is totally fine, right?

Reply:
yeah, but they could remove the RemoteUploaderDownloader or remove the uri prefix?

Reply:
yeah, I guess for me its not quite warning level, but up to you, not a big deal.